docs/TypeScript.md

```diff
@@ -2,10 +2,18 @@
 
 <a id="typescript"></a>
 ## TypeScript
-Fastify is shipped with the a typings file so it should "just work" when used in a TypeScript application.
+Fastify is shipped with the a typings file so it should "just work" when used in a TypeScript application.<br/>
+*You should also install `@types/node` and `@types/pino`.*<br/>
+`npm i @types/node @types/pino --save`
```

Comment:
I assume this is to reduce bloat, but it might be confusing for folks using TypeScript since the versions aren't specified here whereas if included in package.json it would "just work" as claimed.

Reply:
How about we rephrase it as:

Fastify is shipped with the a typings file, but it still require dependent types:

```
npm i @types/node @types/pino@4 --save
```

We might still ship `@types/pino` as a dependency, I'm not 100% sold on that. I would actually prefer if pino shipped with its own typings file.